src/client.ts

```diff
@@ -166,3 +164,28 @@ class DeploymentsClient {
     return resp.data;
   }
 }
+
+class ArchivesClient {
+  /** @internal */
+  private axios: AxiosInstance;
+
+  /** @internal */
+  constructor(axios: AxiosInstance) {
+    this.axios = axios;
+  }
+
+  /**
+   * Fetches a repository snapshot for the specified git revision as a gzipped tarball.
+   *
+   * @see https://docs.bluecanvas.io/reference/checks-api#get-archive
+   */
+  async getTarGzipBlob({ revision }: ArchivesGetTarGzipBlobRequest): Promise<ArchivesGetTarGzipBlobResponse> {
```

Comment:
Will I be able to fetch the `deploymentBranchName` ref from this API? (That would be needed IMO to make it work with DR files in the PMD use case, where we want to process the prepared files with overrides.) in that case, the name `ref` would be more accurate.

Reply:
Yes the API supports branch refs, updated the name to `ref` in b9f1637ccefc13c9f70af1fba9f77455e0da8c48